AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/Mutations.dfy

```diff
@@ -149,6 +166,57 @@ module {:options "/functionSyntax:4" } Mutations {
     return Success(ActiveVerificationHolder.NotDecrypt());
   }
 
+  method VerifyEncryptedHierarchicalKeyForHV2(
+    nameonly item: Types.AwsCryptographyKeyStoreTypes.EncryptedHierarchicalKey,
+    nameonly keyManagerStrategy: KmsUtils.keyManagerStrat,
+    nameonly localOperation: string,
```

Comment:
If we are going to use a `string` for control, which is probably not a best practice, but eh, we MUST put pre-condition the possible values for `localOperation`.

Something like:
```dafny
requires localOperation == "ApplyMutation" || localOperation == "InitializeMutation"
```
Otherwise, it is trivial for a future refactor to break this.
A `type` could also work, and fix this sort of code everywhere.

Reply:
Done

Reply:
Adding a type is a probably a best option but its for another day as its done in many places and we don't have enough time for it.